cddl_gen/cddl_gen.py

```diff
@@ -190,6 +191,8 @@ def __init__(self, default_max_qty, my_types, base_name=None):
         self.my_types = my_types
         self.default_max_qty = default_max_qty  # args.default_max_qty
         self.base_name = base_name  # Used as default for self.get_base_name()
+        # Stem which can be used when generating an id.
+        self.base_stem = base_stem.replace("-", "_") if base_stem else base_stem
```

Comment:
I don't get this. If it is not `None`, then we use its value (with replaced `-`), but if its `None` then we set it to `None` is that the logic here? IMO it would be more clear to state `x.replace() if x is not None else None`

Reply:
Agreed with @hakonfam . Unless `base_stem=None` is evaluated as `None` explicitly down the line, IMO `base_stem=''` would be more representative of the fact that this expected to be and will be operated on as a string if given. Eventually adding type hints would further clarify this, but for now, as a default values go `None` is best avoided unless it makes sense as "not yet assigned".